pkg/ccl/importccl/import_stmt_test.go

```diff
@@ -2174,6 +2174,25 @@ func TestImportIntoCSV(t *testing.T) {
 		)
 	})
 
+	// Tests that we can import into the table even if the table has columns named with
+	// reserved keywords.
+	t.Run("cols-named-with-reserved-keywords", func(t *testing.T) {
+		sqlDB.Exec(t, `CREATE TABLE t ("select" INT PRIMARY KEY, "from" INT, "timestamp" STRING NOT NULL)`)
```

Comment:
i think one or two reserved words if probably fine but i do think we should have a case that requires quoting for special chars, case preservation, etc. e.g. replace timestamp with `Some c,o.l-'Name'`